internal/server/v2/observation/golden/contained_in_latest/Country.json

```diff
@@ -5,27 +5,18 @@
         "country/ABW": {
           "ordered_facets": [
             {
-              "facet_id": "3981252704",
+              "facet_id": "3114843463",
               "observations": [
                 {
-                  "date": "2021",
-                  "value": 106537
+                  "date": "2011",
+                  "value": 108141
                 }
               ]
             }
           ]
         },
         "country/AFG": {
           "ordered_facets": [
-            {
```

Comment:
TODO:  why do we lose this facet?  What does this point to?  @shifucun

Reply:
That is WDI I think - https://github.com/datacommonsorg/mixer/blob/eeb6753abebd05ce32d1c564d7bd2473fc70e754/internal/server/v2/observation/golden/contained_in_latest/Country.json#L5680-L5684C7

Losing it doesn't seem right

Reply:
I think this is the same issue as missing ObsCollection cache @n-h-diaz (b/318430603)